docs/sources/developers/http_api/team_sync.md

```diff
@@ -1,5 +1,6 @@
 ---
 aliases:
+  - ../../http_api/external_group_sync/
   - ../../http_api/team_sync/
```

Comment:
I believe you will also want to redirect from the old page location `/docs/grafana/<VERSION>/developers/http_api/external_group_sync/`.

```suggestion
  - ./external_group_sync/
```